csrc/velox/column.cpp

```diff
@@ -623,6 +624,12 @@ velox::variant pyToVariant(const pybind11::handle& obj) {
     return out;
   }
 
+  if (py::isinstance<functions::GPT2BPEEncoder>(obj)) {
+    out = velox::variant::opaque(
+        obj.cast<std::shared_ptr<functions::GPT2BPEEncoder>>());
+    return true;
```

Comment:
you should return `out` instead of `true`. It expects a variant not a boolean :)

Reply:
or, just simply do 

```c++
return velox::variant::opaque(
        obj.cast<std::shared_ptr<functions::GPT2BPEEncoder>>());
```

Reply:
This did the trick thank you!